sale_stock_picking_blocking/tests/test_sale_stock_picking_blocking.py

```diff
@@ -35,14 +35,15 @@ def setUp(self):
         # Create Sales order and lines:
         so_dict = {
             "partner_id": self.env.ref("base.res_partner_1").id,
+            "name": "Test Sale Delivery Block",
```

Comment:
@kittiu I see you've migrated the module and before [this migration](https://github.com/OCA/sale-workflow/commit/4a38fcf20fcf55f115b36ccdf764ff48c15e91ca#diff-2b7f646de13a325a8564bc11e174f489a505e97bb29a567bbdbf576e222cb19e) a sequence was used.

IMO when we create the order first we should not trigger any revision machinery. The `unrevisioned_name` should probably be a computed stored field and depend on `name`.

What do you think?

Reply:
FTR an issue that might need an answer https://github.com/OCA/sale-workflow/issues/1116

Reply:
Noted, let me try to understand this. You are saying that, this PR need to add this line,
https://github.com/nilshamerlinck/sale-workflow/blob/14.0-fix-sale_stock_picking_blocking-tests/sale_stock_picking_blocking/tests/test_sale_stock_picking_blocking.py#L43

Because without it, test error result from module sale_order_revision.

Correct?